src/controllers/auth.controller.ts

```diff
@@ -51,6 +52,15 @@ export class AuthController {
     return { token };
   }
 
+  @Post('/facebook')
+  async authenticateFacebook(
+    @Body() facebookData: FacebookLoginDTO
+  ) {
+    const user = EntityMapper.mapTo(User, facebookData);
+    const token = await this.sessionService.authenticateFacebook(user);
+    return { token };
```

Comment:
In this case you could simply put 

```typescript
return this.sessionService.authenticateFacebook(user);
```

Because if I understood well...that assignation on line 60 is pretty useless, right?